nginx.rb

```diff
@@ -24,6 +24,16 @@ def vhost_link
   def upstream_name
     "#{domain}.upstream"
   end
+
+  def cert_name
+    # Grab the TLD from the domain (e.g. dw.theconversation.com ->
+    # theconversation.com).
+    cert_domain = domain.to_s
+      .match(/([^.]+)\.([^.]+)$/).to_s
+      .gsub('.', '_')
+
+    "STAR_#{cert_domain}"
```

Comment:
just curious why the `STAR_` prefix?

Reply:
wildcard?

Reply:
Yeah. I just kept things consistent with the DNSimple naming. They call it `STAR_theconversation_com`, etc.

Reply:
ah nice one 👍